components/identity-mgt/org.wso2.carbon.identity.mgt/src/main/java/org/wso2/carbon/identity/mgt/IdentityMgtEventListener.java

```diff
@@ -814,8 +814,7 @@ public boolean doPreSetUserClaimValues(String userName, Map<String, String> clai
                 try {
                     identityDataStore.store(identityDTO, userStoreManager);
                 } catch (IdentityException e) {
-                    throw new UserStoreException(
-                            "Error while saving user store data for user : " + userName, e);
+                    throw new UserStoreException(e.getMessage(), e);
```

Comment:
What is the reason for using e.getMessage(), I think since it is not the praise we usually use, if there is an specific reason, it would be better to add a comment, otherwise someone can replace this with a message like previous one.